build/x86_64_debian/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM debian:7
```

Comment:
This file name doesn't declare that it's for debian7, should we be consistent with what we're doing for centos?

Reply:
I tried to keep the containers as closely linked to the repos where the packages are published. There's just one generic debian repo for all logging agent version, which is why I didn't explicitly call out "debian7" or "wheezy" in the container name or directory.